src/components/Modals/AssetSelectorModal/ChainSectionFooter.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2021 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import * as React from 'react';
+import styled from 'styled-components/native';
+import { useTranslation } from 'translations/translate';
+
+// Components
+import Icon from 'components/core/Icon';
+import Text from 'components/core/Text';
+
+// Utils
+import { useThemeColors } from 'utils/themes';
+import { appFont, fontStyles, spacing } from 'utils/variables';
+
+type Props = {|
+  showMore: boolean,
+  onPress: () => mixed,
+|};
+
+function ChainSectionFooter({ showMore, onPress }: Props) {
+  const { t } = useTranslation();
+  const colors = useThemeColors();
+
+  return (
+    <Container>
+      {showMore && (
```

Comment:
Not fully get it, if there's no `showMore` then simply `Container` with extra space is added?

Reply:
Extra space is always needed, because it separate this section from the next one.

Reply:
Ok cool, just the fact that the space is produced from empty content components makes me consider if it can be moved somewhere outside?

Reply:
Not really, as the space is needed after every section of the section list. If moved to the item, it would be repeated after each item, if moved to the header it would be also applied before first element. Hence the reasonable solution is to put in the section footer as I've done here.